main/zoe/api/zoe.md

```diff
@@ -3,217 +3,204 @@
 ```js
-const {
-  instanceHandle,
-  installationHandle,
-  publicAPI,
-  terms
-} = await E(zoe).getOffer(offerHandle);
+const instance = await E(zoe).getInstance(invitation);
 ```
+      
+## E(zoe).getInstallation(invitation)
+- `invitation` `{Invitation}`
+- Returns: `{Promise<Installation>}`
 
-## E(zoe).getCurrentAllocation(offerHandle, sparseKeywords)
-- `offerHandle` <router-link to="/glossary/#handle">`{Array <Handle>}`</router-link>
-- `sparseKeywords` sparseKeywords is an array of string keywords, which may be a subset of allKeywords.
-- Returns: <router-link to="/zoe/api/records.html#offer-record">`{<AmountKeywordRecord>}`</router-link>
-
-Get the amounts associated with the sparseKeywords for the offer.
-
+Returns a `Promise` for the contract `installation` the `invitation`'s contract `instance` uses.
 ```js
-const { foo, bar } = E(zoe).getCurrentAllocation(offerHandle, ['foo', 'bar']);
+const installation = await E(zoe).getInstallation(invitation);
 ```
-
-## E(zoe).getCurrentAllocations(offerHandles, sparseKeywords)
-- `offerHandles` <router-link to="/glossary/#handle">`{Array <Handle>}`</router-link>
-- `sparseKeywords` sparseKeywords is an array of string keywords, which may be a subset of allKeywords.
-- Returns: <router-link to="/zoe/api/records.html#amount-keyword-record">`{ Array AmountKeywordRecord>}`</router-link>
-
-Get the amounts associated with the sparseKeywords for the offers.
-
-
-## E(zoe).getOfferNotifier(offerHandle)
-- `offerHandle` <router-link to="/glossary/#handle">`<Handle>`</router-link>
-- Returns: a <router-link to="/glossary/#notifier">notifier</router-link> for the offer.
+## E(zoe).startInstance(installation, issuerKeywordRecord, terms)
+- `installation` `{Installation}`
+- `issuerKeywordRecord` `{IssuerKeywordRecord}`
+- `terms` `{Object}`
+- Returns: `{Promise<StartInstanceResult>}`
+
+Create an `instance` of the installed smart contract (specified by 
+the `installation` argument). You must also specify the 
+instance's `issuerKeywordRecord` and `terms` for the contract 
+(as key-value pairs). 
+
+The `issuerKeywordRecord` is a record mapping string names (keywords) 
+to `issuers`, such as `{ Asset: quatlooIssuer}`. Keywords must begin 
+with a capital letter and must be ASCII. Parties to the contract will 
+use the keywords to index their proposal and their payments. 
+
+`terms` are values used by this contract instance, such as the 
+number of bids an auction will wait for before closing. These values may
+be different for different instances of the same contract, but the contract
+defines what variables need their values passed in as `terms`. 
+
+It returns a `promise` for a `StartInstanceResult` object. The object consists of:
+- `creatorFacet` `{any}`
+- `publicFacet` `{any}`
+- `instance` `{Instance}`
+- `creatorInvitation` `{Payment | undefined}`
+
+A `publicFacet` and a `creatorFacet` are objects available via Zoe to anyone knowing 
```

Comment:
```suggestion
A `publicFacet` is an object available via Zoe to anyone knowing 
```

The `creatorFacet` is only available in this return value. The contract designer should use it to encapsulate things that the contract runner might not want to share, or might want to control the distribution of. The party who starts the contract should carefully consider the impact before sharing access to the creatorFace.